src/core/messages/DraftBotBroadcastValidationMessage.ts

```diff
@@ -186,15 +186,16 @@ export class DraftBotBroadcastValidationMessage extends DraftBotEmbed {
 	 */
 	private async manageDenyReaction(user: User) {
 		if (this._interaction.user.id === user.id) {
-			await sendErrorMessage(user, this._interaction.channel, this._language, this._translationModule.errorBroadcastCancelled, true);
+			sendErrorMessage(this._interaction.user, this._interaction, this._language, this._translationModule.errorBroadcastCancelled, true);
```

Comment:
boarf, ici garde user tout court, c'est le même
```suggestion
			sendErrorMessage(user, this._interaction, this._language, this._translationModule.errorBroadcastCancelled, true);
```